src/js/app.js

```diff
@@ -0,0 +1,55 @@
+$(document).ready(function() {
+
+  $('#searchBtn').on('click', function(event) {
+    event.preventDefault();
+
+     $.get('https://www.reddit.com/search.json', {
+      q: valInput()
+    }).done(function(taco) {
+      intoArray(taco);
+    }).fail(function(taco) {
+      console.log('An error occured');
+    })
+
+  })
+
+  $('#resetBtn').click(function() {
+    location.reload();
```

Comment:
I'll mention this to everyone, but `location.reload` isn't always recommended because it requires another request from the server or cache. It's faster to reset everything via JavaScript